configs/templates/skylab-dev/spack.yaml

```diff
@@ -36,17 +28,17 @@ spack:
     - ecflow@5
     - eckit@1.20.2
     - ecmwf-atlas@0.31.1
-    - ectrans@1.1.0
+    - ectrans@1.2.0
     - eigen@3.4.0
     - esmf@8.3.0b09+pio
     # DH* fake version number
     #- ewok@0.0.1
     - fckit@0.9.5
     # DH* fake version number
-    - fiat@1.0.0
+    - fiat@1.1.0
     # Do not request specific version of flex - this leads to duplicate packages being built
     #- flex@2.6.4
-    - fms@2022.02
+    - fms@2022.04
```

Comment:
Does this need the `constants=GFS` added (like what is done in `configs/templates/ufs-weather-model-static/spack.yaml`)?

Reply:
Ditto